source/faq.txt

```diff
@@ -113,33 +113,31 @@ is on the other side of the planet and has a latency of 10000
 milliseconds, setting the ``connectTimeoutMS`` to anything lower will
 prevent the driver from ever connecting to that member.
 
-Should I Use "socketTimeoutMS" as a Way of Preventing Long-Running Operations From Slowing Down the Server?
--------------------------------------------------------------------------------------------------------------
+What Happens to My Operations if My Client Disconnects?
+-------------------------------------------------------
 
-No, you should **not** use ``socketTimeoutMS`` to end operations that
-may run for too long and slow down the application. Attempting to do so
-may not achieve the intended result.
+Starting in {+mdb-server+} version 4.2, the server terminates
+:manual:`certain operations </release-notes/4.2/#client-disconnection>` such as
+aggregations and find operations if the client disconnects before it
+successfully executes an operation.
 
-Closing the socket causes a reconnect of the driver's connection pool,
-introducing latency to any other queued up operations. Chronically slow
-operations will, therefore, cause a large number of reconnect requests,
-negatively impacting throughput and performance.
+Some operations, including all write operations, continue to run on the
```

Comment:
Suggestion:

I think a transition signal word would help the reader immediately understand that this paragraph introduces the exception cases.

```suggestion
Other operations, such as write operations, continue to run on the
```